install/streams.csv

```diff
@@ -2067,8 +2067,16 @@ RTZ,10,H,H,100,false,false,false,2011-08-04T00:00:10Z,9999-01-01T00:00:00Z
 RTZ,20,B,N,50,false,false,false,2011-05-05T00:05:00Z,2020-06-16T00:00:00Z
 RTZ,20,H,N,200,false,false,true,2011-05-05T00:05:00Z,2020-08-13T01:19:00Z
 RTZ,20,H,N,200,false,false,false,2020-08-13T01:26:00Z,9999-01-01T00:00:00Z
-RU001,01,,,-600,false,false,false,2018-05-03T03:50:00Z,9999-01-01T00:00:00Z
-RU001,02,,,-600,false,false,false,2019-03-20T23:00:00Z,9999-01-01T00:00:00Z
+RU001,01,,,-3600,false,false,false,2009-04-15T02:00:00Z,2010-04-09T00:00:00Z
+RU001,01,,,-900,false,false,false,2010-04-13T12:15:00Z,2010-05-15T04:45:00Z
+RU001,01,,,-900,false,false,false,2011-04-12T21:00:00Z,9999-01-01T00:00:00Z
+RU001,01,,,-450,false,false,false,2010-05-15T04:52:30Z,2011-04-12T20:52:30Z
```

Comment:
minor thing, but shall all these RU001,01 entries be in chronological order? (is it passing the test, so ok if not changed)
```
elidana@hutl24201:~/tmp/delta-2/install$ grep RU001,01 streams.csv | sort -t, -nk 10
RU001,01,,,-3600,false,false,false,2009-04-15T02:00:00Z,2010-04-09T00:00:00Z
RU001,01,,,-900,false,false,false,2010-04-13T12:15:00Z,2010-05-15T04:45:00Z
RU001,01,,,-450,false,false,false,2010-05-15T04:52:30Z,2011-04-12T20:52:30Z
RU001,01,,,-900,false,false,false,2011-04-12T21:00:00Z,9999-01-01T00:00:00Z
```

Reply:
The order is required for the file to pass the tests https://github.com/GeoNet/tickets/issues/16263